### DISCUSSION TITLE
Better handling for symlinks when deploying

### DISCUSSION THREAD

**Discussion by baked-dev** [vercel/vercel#8525] [Ideas]
Hey,

I'm currently having the issue that I can't deploy our next.js project through CLI when using pnpm as the package manager because i end up with more than 15000 files in my output folder. Most of the files in the output are just symlinks but i assume the CLI resolves them and uploads entirety of the same file multiple times. It would be nice if the platform supported uploading symlinks and then not consider those towards the 15000 deployment file limit.

**Comment by amyegan** [comment]
Thanks for reaching out @baked-dev! Can you tell me more about your Project setup? If you have a `pnpm-lock.yaml` checked in or `pnpm install` set as your Install Command, then it should be using pnpm. But if you don't have one of those in your Project, then it could be defaulting to yarn. You can read more about how install commands are automatically selected here: [vercel.com/docs/concepts/deployments/configure-a-build#install-command](https://vercel.com/docs/concepts/deployments/configure-a-build#install-command)

If it still isn't working, can you share your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example)? If you can share a reproduction, then I can dig in a bit more to find the source of the problem.
